internal/uvm/create_wcow.go

```diff
@@ -193,15 +195,22 @@ func CreateWCOW(ctx context.Context, opts *OptionsWCOW) (_ *UtilityVM, err error
 			BandwidthMaximum: opts.StorageQoSBandwidthMaximum,
 		}
 	}
-
 	uvm.scsiLocations[0][0].hostPath = doc.VirtualMachine.Devices.Scsi["0"].Attachments["0"].Path
 
 	fullDoc, err := mergemaps.MergeJSON(doc, ([]byte)(opts.AdditionHCSDocumentJSON))
```

Comment:
We should take a moment to remove this merging code (as a separate commit that we can PR immediately). As far as I know it's not used anywhere and it just makes things more complicated.